nebula_scripts/Build_cabal_and_project.py

```diff
@@ -0,0 +1,105 @@
+# This script is used to modify the cabal file by change or add a build depends and ghc-option
+# This script also builds a cabal.project  
+# Three arguments needed:
+# first argument is the python file
+# second argument is the directory that contain all the package that contain rules
+# third argument is the location of g2 in one's computer so we can 
```

Comment:
Missing part of the sentence here?  "so we can" what?

Reply:
So we can run Nebula on those packages. Sorry for that mistake